custom-recipes/timeseries-forecast-1-train-evaluate/recipe.py

```diff
@@ -1,8 +1,14 @@
 # -*- coding: utf-8 -*-
+try:
```

Comment:
Can we put that in a lib instead of recipe.py? Also, we shouldn't try this if the GPU is not activated by the user.
Last thing, is there a more specific error class than OSError we can catch and re-raise? With OSError, we are not sure it is indeed a CUDA-related problem